### ISSUE TITLE
How to get the class prediction probability/confidence of each pixel?

### ISSUE THREAD

**Issue by ZF-AR** [CSAILVision/semantic-segmentation-pytorch#120]
Thank you!

**Comment by hangzhaomit** [comment]
Apply softmax after this:
https://github.com/CSAILVision/semantic-segmentation-pytorch/blob/master/eval.py#L68
